cdi/src/main/java/org/togglz/cdi/spi/CDIBeanFinder.java

```diff
@@ -67,6 +68,21 @@ private BeanManager getBeanManager(ServletContext servletContext) {
             beanManager = (BeanManager) servletContext.getAttribute(SERVLET_CONTEXT_ATTR_WELD_1_1);
         }
 
+        // try Weld 5.1.x servlet context attribute
+        if (beanManager == null && servletContext != null) {
+            beanManager = (BeanManager) servletContext.getAttribute(SERVLET_CONTEXT_ATTR_WELD_5_1);
+        }
+
+        // try jakarta servlet context attribute
+        if (beanManager == null && servletContext != null) {
+            beanManager = (BeanManager) servletContext.getAttribute("jakarta.enterprise.inject.spi.BeanManager");
```

Comment:
could you please extract this string to a constant as well?

Reply:
I removed the block, this did not work anyway.